pysat/_constellation.py

```diff
@@ -59,6 +59,10 @@ class Constellation(object):
         Input dict containing dicts of inputs for `custom_attach` method inputs
         that may be applied to all instruments or at the Constellation-level or
         None (default=None)
+    use_header : bool
```

Comment:
Perhaps introduce `**kwargs` here instead?  This will be a longer term solution for future changes and also improve integration with `instruments`.